posthog/tasks/usage_report.py

```diff
@@ -63,6 +64,14 @@
 QUERY_RETRY_DELAY = 1
 QUERY_RETRY_BACKOFF = 2
 
+USAGE_REPORT_TASK_KWARGS = dict(
+    queue=CeleryQueue.USAGE_REPORTS.value,
```

Comment:
@raquelmsmith just FYI - modified this to have each task use the queue. It doesn't work to only have the parent task on its own. The task that get created also need to specify their intended queue

Reply:
Oh that makes sense. Thanks!